lib/Twig/Template/DisplayCallGraphTemplate.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2010 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Displays the call graph (templates names and block names) in a visual way directly in the generated HTML.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+abstract class Twig_Template_DisplayCallGraphTemplate extends Twig_Template
+{
+    protected $templateStart = '<div style="border: 1px solid rgba(240, 181, 24, 0.3); margin: 5px;"><span style="background-color: rgba(240, 181, 24, 0.3); color: black; font-family: monospace;">Template "%s"</span>';
+    protected $templateEnd = '</div>';
+    protected $blockStart = '<div style="border: 1px solid rgba(100, 189, 99, 0.2); margin: 5px;"><span style="background-color: rgba(100, 189, 99, 0.2); color: black; font-family: monospace;">Block "%s"</span>';
```

Comment:
IMO extensibility to change all those values is better implemented using protected getters. so to change it, people can overwrite the method. then we do not need to deal with setters and properties and it's easier to maintain bc.